apps/api/src/.example.env

```diff
@@ -78,3 +78,6 @@ API_RATE_LIMIT_MAXIMUM_FREE_GLOBAL=
 API_RATE_LIMIT_MAXIMUM_UNLIMITED_TRIGGER=
 API_RATE_LIMIT_MAXIMUM_UNLIMITED_CONFIGURATION=
 API_RATE_LIMIT_MAXIMUM_UNLIMITED_GLOBAL=
+
+STRIPE_API_KEY=
+STRIPE_CONNECT_SECRET=
```

Comment:
Could we also list these under https://github.com/novuhq/novu/blob/next/apps/api/src/types/env.d.ts#L25 for type safety?

Reply:
We should also add these env-vars to the env-validator with undefined default values:
https://github.com/novuhq/novu/blob/next/apps/api/src/config/env-validator.ts#L84